airlock/views/request.py

```diff
@@ -693,21 +693,20 @@ def requests_for_workspace(request, workspace_name: str):
     requests_for_workspace = bll.get_requests_for_workspace(
         workspace_name, request.user
     )
+    requests_for_workspace_unfiltered = requests_for_workspace
```

Comment:
Tiny nit: this line is unnecessary if you build the `status_choice` dict here, before you do the `if request_filter` block